core/new-gui/src/app/workspace/service/joint-ui/joint-ui.service.ts

```diff
@@ -769,12 +769,12 @@ export class JointUIService {
         ry: "5px",
       },
       "rect.boundary": {
-        fill: "rgba(0,0,0,0)",
-        width: this.DEFAULT_OPERATOR_WIDTH + 50,
-        height: this.DEFAULT_OPERATOR_HEIGHT + 100,
+        fill: "rgba(0, 0, 0, 0)",
```

Comment:
is it possible to make this a variable so that we color the operators (green for addition), (orange for modifications), and (red for removal)